src/nix/develop.cc

```diff
@@ -437,7 +437,7 @@ struct CmdDevelop : Common, MixEnvironment
             std::vector<std::string> args;
             for (auto s : command)
                 args.push_back(shellEscape(s));
-            script += fmt("exec %s\n", concatStringsSep(" ", args));
+            script += concatStringsSep(" ", args);
```

Comment:
Why is this needed?

Reply:
`exec` isn't needed *anymore*, so I thought I may as well simplify the code (since it's not being used for `--phase` either). Before, it was necessary for exiting the then-interactive shell.

Reply:
I don't have a strong opinion on this, but it seems cleaner to keep it, if only to avoid having a useless bash process lying around while the command is running

Reply:
Yeah, why not. Reverted.